assume-aws-sso-role

```diff
@@ -118,6 +120,15 @@ role_info_from_profile() {
   set_account_and_role_from_role_arn
 }
 
+get_partition_from_region() {
+  if [[ ${aws_region} = cn-* ]] then
```

Comment:
I believe there is a syntax error here. There should be a `;` before `then`.

Reply:
You are absolutely right and it should be fixed now.